composer.json

```diff
@@ -58,7 +58,7 @@
   },
   "scripts": {
     "cs:check": "php-cs-fixer fix -v --diff --dry-run",
-    "cs:fix": "php-cs-fixer fix -v --diff",
+    "cs:fix": "php-cs-fixer fix --config .php-cs-fixer.php -v --diff",
```

Comment:
Do we really need this? As `.php-cs-fixer.php` is the default file name, I don't think it should be necessary. Either way: it should be consistent, so remove it here or add it for `cs:check` if it's really necessary.

Reply:
Ah, I see that, `bin/php-cs-fixer`, also has it. But I assume we can remove it there, too.

Reply:
Yes, but to be honest - bin/php-cs-fixer run does nothing...don't know why

Reply:
Did you call `bin/php-cs-fixer run`? I think it is `bin/php-cs-fixer fix`.